utils/_context/_scenarios.py

```diff
@@ -999,7 +999,7 @@ class scenarios:
         appsec_enabled=True,
         weblog_env={
             "DD_EXPERIMENTAL_API_SECURITY_ENABLED": "true",
-            "DD_TRACE_DEBUG": "true",
+            "DD_TRACE_DEBUG": "false",
```

Comment:
typo ?

Reply:
wanted change as explained in the PR notes.